zproject/settings.py

```diff
@@ -1079,3 +1079,5 @@ def get_secret(key):
 PROFILE_ALL_REQUESTS = False
 
 CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+
+USE_SOCKET = True
```

Comment:
So if I move the setting into the DEFAULT_SETTINGs dict I'm correct in saying it will be initialised to True unless it's changed in /etc/zulip/settings.py?